python/ray/data/tests/test_sql.py

```diff
@@ -265,35 +265,49 @@ def request_get_mock(url, params=None, **kwargs):
         ray.init()
 
         # test query with a table name
-        result = ray.data.read_databricks_tables(
-            warehouse_id=warehouse_id,
-            table="table1",
-            catalog="catalog1",
-            schema="db1",
-            override_num_blocks=5,
-        ).to_pandas()
-
+        result = (
+            ray.data.read_databricks_tables(
+                warehouse_id=warehouse_id,
+                table="table1",
+                catalog="catalog1",
+                schema="db1",
+                override_num_blocks=5,
+            )
+            .to_pandas()
+            .sort_values("c1")
+            .reset_index(drop=True)
```

Comment:
This test implicitly assumed that read tasks complete in order, which is true for `LazyBlockList` but not for the new streaming executor.